joint_trajectory_controller/include/joint_trajectory_controller/trajectory.hpp

```diff
@@ -149,6 +149,7 @@ class Trajectory
   rclcpp::Time time_before_traj_msg_;
   trajectory_msgs::msg::JointTrajectoryPoint state_before_traj_msg_;
 
+  size_t prev_start_idx_;
```

Comment:
Would be make more sense?
```suggestion
  size_t previously_start_index_;
```